vmdb/app/models/storage.rb

```diff
@@ -67,7 +67,7 @@ class Storage < ActiveRecord::Base
   SUPPORTED_STORAGE_TYPES = ["VMFS", "NFS"]
 
   def miq_proxies
-    MiqProxy.find(:all).select { |p| p.storages.include?(self) }
+    MiqProxy.all.select { |p| p.storages.include?(self) }
```

Comment:
This should have a `.includes(:storages)` (if storages are a real or virtual relationship here)